### ISSUE TITLE
Conceptual Issue: Cryptocurrency? "Wages for Facebook"?

### ISSUE THREAD

**Issue by marxzuckerburg** [ComradOrg/Comrad#27]
"And also, will this have a cryptocurrency? I don't really see the reason, unless it is some reward for hosting nodes (like LokiNet does) or it is needed for some PoW stuff, but anyways if it will, I would recommend forking ARRR since IMO it is the most promising anonymous crypto out there." -LJS

"This is really interesting. Touches on ideas like 'Wages For Facebook', built on that older Silvia Federici feminist argument of 'Wages For Housework'. It would be interested if posting -- contributing content, which benefits everyone, and which is a form of labor -- could somehow produce 'value' to the network that is then shared back with users, not necessarily according to who posted most but in dialogue with that. The currency ('points'? 'kudos'?) would of course be fake (custom made, costing nothing in real money), at least at first -- but if people donate to the entire network then maybe some cryptocurrency credits are partitioned out to people? Would love to keep thinking about that." -MZ

Any other thoughts on this, on the role of cryptocurrency in an alternative social(ist) network?

**Comment by gary-host-laptop** [comment]
I think it's totally unnecessary, the only use case I see it's literally as a reward for hosting nodes, but that's not how Komrade works, maybe getting rewards depending on your posts or followers or activity sounds good, but it's something really hard to implement, and who would be mining that? It just makes the project sound a bit shady, since there are a lot of these kind of stuff where devs usually benefit from that (premined crypto, dev rewards, etc(.

**Comment by marxzuckerburg** [comment]
Yeah, just as the Wages For Housework and Wages For Facebook movements are politically circumspect in certain ways -- do we really want to *further* transactionalize our social relations? -- it's not clear how introducing currency to mundane social interactions is a good idea. I would howeveer like to keep thinking about how digital socialist tools can lead to providing financial support to people abandoned by the state, like through digital wealth redistirbution mechanisms which might supplement failed graduated income tax plans. But that's another topic maybe
